lsp-metals.el

```diff
@@ -117,7 +117,9 @@ more customizations like using environment variables."
   (erase-buffer)
   (insert html)
   (shr-render-region (point-min) (point-max))
-  (setq-local buffer-read-only t))
+  (goto-char (point-min))
+  (view-mode 1)
+  (setq view-exit-action 'kill-buffer))
```

Comment:
setq-local?

Reply:
`view-exit-action` automatically becomes buffer-local when set. So it should be fine unless you think it's clearer to use `setq-local` explicitly :)

Reply:
That's true.